File: Practice/21/C++/21.cpp
#include <iostream>
using namespace std;
double BMI(double weight, double height)
{
    double x = weight / (height * height);
    return x;
}
void printBMI(double x)
{
    if (x < 18.5)
        cout << "Underweight";
    if (x >= 18.5 && x < 25.0)
        cout << "Normal weight";
    if (x >= 25.0 && x < 30.0)
        cout << "Overweight";
    if (x >= 30.0)
        cout << "Obesity";
}
int main()
{
    double height, weight, x;
    x = 0;
    cin >> weight;
    cin >> height;
    BMI(weight, height);
    printBMI(x);

}

File: Practice/21/Python/21.py
def bmi(weight: float, height: float) -> float:
    return weight / (height * height)
def print_bmi(bmi: float) -> float:
    if (bmi < 18.5):
        print('Underweight')
    if (18.5 <= bmi < 25):
        print("Normal weight")
    if (25 <= bmi < 30):
        print("Overweight")
    if (bmi >= 30):
        print("Obesity")
print("Введите ваш вес и рост")  
w, h = map(float,input().split())
BMI = bmi(w, h)
print_bmi(BMI)
